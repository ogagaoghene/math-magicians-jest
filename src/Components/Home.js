import React from 'react';
import '../css/home.css';

const Home = () => (
  <>
    <div className="grid__wrapper">
      <h1>Welcome to Our Page!</h1>
      <p>
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Amet esse
        aperiam reprehenderit repudiandae quod vitae? Eius perspiciatis expedita
        porro quos accusamus odio non, consequatur provident enim, assumenda
        distinctio aperiam dolores.
      </p>
      <p>
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Amet esse
        aperiam reprehenderit repudiandae quod vitae? Eius perspiciatis expedita
        porro quos accusamus odio non, consequatur provident enim, assumenda
        distinctio aperiam dolores.
      </p>
      <p>
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Amet esse
        aperiam reprehenderit repudiandae quod vitae? Eius perspiciatis expedita
        porro quos accusamus odio non, consequatur provident enim, assumenda
        distinctio aperiam dolores.
      </p>
      <p>
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Amet esse
        aperiam reprehenderit repudiandae quod vitae? Eius perspiciatis expedita
        porro quos accusamus odio non, consequatur provident enim, assumenda
        distinctio aperiam dolores.
      </p>
    </div>
  </>
);

export default Home;
