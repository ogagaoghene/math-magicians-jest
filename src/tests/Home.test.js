import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Components/Home'

describe("Snapshot testing", () => {
    it("Check if Home component is rendered", () => {
        const tree = renderer.create(<Home />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})