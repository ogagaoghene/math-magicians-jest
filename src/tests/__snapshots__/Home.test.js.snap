// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing Check if Home component is rendered 1`] = `
<div
  className="grid__wrapper"
>
  <h1>
    Welcome to Our Page!
  </h1>
  <p>
    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Amet esse aperiam reprehenderit repudiandae quod vitae? Eius perspiciatis expedita porro quos accusamus odio non, consequatur provident enim, assumenda distinctio aperiam dolores.
  </p>
  <p>
    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Amet esse aperiam reprehenderit repudiandae quod vitae? Eius perspiciatis expedita porro quos accusamus odio non, consequatur provident enim, assumenda distinctio aperiam dolores.
  </p>
  <p>
    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Amet esse aperiam reprehenderit repudiandae quod vitae? Eius perspiciatis expedita porro quos accusamus odio non, consequatur provident enim, assumenda distinctio aperiam dolores.
  </p>
  <p>
    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Amet esse aperiam reprehenderit repudiandae quod vitae? Eius perspiciatis expedita porro quos accusamus odio non, consequatur provident enim, assumenda distinctio aperiam dolores.
  </p>
</div>
`;
